src/Microsoft.AspNet.SignalR.Core/Infrastructure/DispatchingTaskCompletionSource.cs

```diff
@@ -26,11 +26,6 @@ public void SetException(Exception exception)
             TaskAsyncHelper.Dispatch(() => _tcs.SetException(exception));
```

Comment:
Should DbOperation.ExecuteNonQueryAsync and SqlReceiver also be updated to use DispatchingTaskCompletionSource?